serverless/admin_guide/serverless-ha.adoc

```diff
@@ -18,7 +18,7 @@ High availability (HA) functionality is available by default on {ServerlessProdu
 
 * Knative Serving: `activator`, `autoscaler`, `autoscaler-hpa`, `controller`, `webhook`, `kourier-control`, `kourier-gateway`.
 * Knative Eventing: `eventing-controller`, `eventing-webhook`, `imc-controller`, `imc-dispatcher`, `mt-broker-controller`, `sugar-controller`.
-* Knative Kafka: `kafka-ch-controller`, `kafka-controller-manager`, `kafka-webhook`.
+* Knative Kafka: `kafka-controller`, `kafka-webhook-eventing`.
 
 These components are configured with two replicas by default.
```

Comment:
This whole section is actually updated by a different PR, so this isn't needed in this PR. Will add a link later.

Reply:
Link: https://github.com/openshift/openshift-docs/pull/46468

PR is ready to merge, can be merged after this one.